src/components/Pill.svelte

```diff
@@ -5,15 +5,13 @@
 
 <style>
   .pill {
-    @apply inline-block;
-    @apply border-transparent;
-    @apply rounded-full;
-    @apply px-2;
-    @apply py-0;
-    @apply text-sm;
-    @apply leading-5;
-    @apply text-white;
-    @apply font-medium;
+    display: inline-block;
+    border-radius: 9999px;
+    border-color: transparent;
+    padding: 0 $spacing-sm 0 $spacing-sm;
+    @include text-body-sm;
```

Comment:
I wonder if we should move these `@include` directives to the top of the declaration. Might make it a little more clear.